gdc_client/upload/client.py

```diff
@@ -461,15 +463,15 @@ def upload_parts(self):
             manager = Manager()
             self.ns = manager.Namespace()
             self.ns.completed = 0
-        part_amount = int(math.ceil(self.file_size / float(self.part_size)))
+        part_amount = int(math.ceil(self.file_size / float(self.upload_part_size)))
         self.total_parts = part_amount
         self.pbar = ProgressBar(
             widgets=[Percentage(), Bar()], maxval=self.total_parts).start()
         try:
             for i in xrange(part_amount):
-                offset = i * self.part_size
+                offset = i * self.upload_part_size
                 remaining_bytes = self.file_size - offset
-                bytes = min(remaining_bytes, self.part_size)
+                bytes = min(remaining_bytes, self.upload_part_size)
```

Comment:
You might want to rename this variable, its same as built in type `bytes`

Reply:
sure